js_basics_HT/task5_3.js

```diff
@@ -0,0 +1,5 @@
+const height = 5 
+const radius = 3
+const SC = (Math.PI*Math.pow(radius,2)*height)
```

Comment:
```js
const cylinderVolume = Math.PI * Math.pow(radius,2) * height
```

Reply:
SC - square cylinder (my bad, I've made a mistake after previous tasks)
does () play any role in this case?

Reply:
does () play any role in this case? - it does not, it's just unnecessary here